kraken-java-sdk/kraken-java-sdk-auth/src/test/resources/application-test-hs512.yaml

```diff
@@ -1,15 +1,14 @@
----
 app:
   security:
     login:
       enabled: true
       jwt:
         issuer: https://kraken.consoleconnect.com/issuer
         key-id: kraken
-        secret: MjRhMjNhNGYtMWUzMS00ZTY4LTlhYzgtMzY5NDcwYzNjNDE4
+        secret: Y2ZmZTdlZmUtMzdiMS00ZGVmLWEzMWItMWNiMGM4ZGQxYWU1
```

Comment:
pls rollback the default token to `MjRhMjNhNGYtMWUzMS00ZTY4LTlhYzgtMzY5NDcwYzNjNDE4`